Performance-Tracing.md

```diff
@@ -26,6 +26,12 @@ The goal of this process is to be able to extract a reduced repro for which you
     You can use `.` as the directory, but it may get cluttered, so a subdirectory is usually preferable.
     In this early version of the feature, you have to use `tsc` specifically — building through a bundler that invokes TypeScipt via the API will not work.
     For best results, make sure this is not an incremental build (i.e. pass `-f` in build mode or `--incremental false` for regular compilation).
+    
+    For example:
+    
+    ```sh
+    tsc -p some_directory --generateTrace some_directory
+    ```
```

Comment:
Seems kind of weird to me, I'd *expect* adding `generateTrace` to any other flags that I use.

Reply:
I think outside of compiler folk, using a TSConfig file is much more prevalent - so lets keep this and get it in